sqlmesh/core/engine_adapter/base.py

```diff
@@ -951,6 +986,11 @@ def create_view(
         if not properties:
             properties = exp.Properties(expressions=[])
 
+        if view_properties:
+            table_type = self._pop_creatable_type_from_properties(view_properties)
+            if table_type:
+                properties.append("expressions", table_type)
```

Comment:
Let's  be consistent and either do the `extend(ensure_list(...))` trick here or change the other sections.